contracts/contracts/sbtc-token.clar

```diff
@@ -0,0 +1,146 @@
+(define-constant err-not-token-owner (err u4)) ;; `tx-sender` or `contract-caller` tried to move a token it does not own.
+(define-constant err-not-protocol-caller (err u5)) ;; `tx-sender` or `contract-caller` is not the protocol caller
```

Comment:
Nit, and I don't have a preference for `err-something` vs `ERR_SOMETHING`, but we should keep things consistent IMO.

Also, I think it'd be good to use the same "number space" pattern we have with other contract's error codes, ie XX4, XX5. For the "not token owner" error we should probably keep 4 to stay in line with the SIP, but otherwise use XXX for non-sip error codes

Reply:
Copy & correct, good catch, addressed.